R/fuzzy_match_mafs.R

```diff
@@ -0,0 +1,119 @@
+#' @title Fuzzy match mafs
+#'
+#' @description Compare two MAFs and identify pairs of rows as likely matches using approximate positions (mostly for indels)
+#'
+#' @details Specify the two mafs in a data frame format to compare as `maf1` and `maf2`. In the generated output, the data
+#' frame is derived from maf1 with two new columns (maf1_ID and maf2_ID). Values in maf1_ID are derived from the maf1.
+#' The values in maf2_ID are derived from maf2 and are NA for any rows that could not be matched to maf2.
+#'
```

Comment:
These lines in the documentation exceed the 80-character-per-line best practice, this violation will result in such lines being truncated in the PDF documents.

Reply:
I've chopped this and modified, same for parameter description - and these lines should be good now 👍 
Thanks!